src/utils/Util.js

```diff
@@ -184,6 +184,23 @@ export function getCubeDescription(cube, changedCards) {
   return `${cardCount} Card ${overridePrefixes}Cube`;
 }
 
+export function userIsDocumentOwner(user, cube) {
+  if (user && cube && cube.owner) {
+    return cube.owner.id === user.id;
+  }
+  
+  return false;
+};
+  
+export function userIsDocumentCollaborator(user, cube) {
+  if (user && cube && cube.collaborators) {
+    return cube.collaborators.some((c) => c.id === user.id);
+  }
+
+  return false;
+};
+
+
```

Comment:
Changing the variable name to doc here
```suggestion
export function userIsDocumentOwner(user, doc) {
  if (user && doc?.owner) {
    return doc.owner.id === user.id;
  }
  
  return false;
};
  
export function userIsDocumentCollaborator(user, cube) {
  if (user && doc?.collaborators) {
    return doc.collaborators.some((c) => c.id === user.id);
  }

  return false;
};

```